test/mocha/10-api.js

```diff
@@ -44,4 +44,76 @@ describe('readiness', () => {
     result.dependencies.ready1.ready.should.equal(true);
     result.dependencies.ready2.ready.should.equal(false);
   });
+  it('should throw error if readiness protocol is not supported',
```

Comment:
This looks like it returns an error, not throws an error.

Reply:
Thanks, updated 4b29d14